reservation_units/tests/test_reservation_unit_export.py

```diff
@@ -0,0 +1,130 @@
+import csv
+import shutil
+from pathlib import Path
+from typing import Any, List
+
+from assertpy import assert_that
+from django.conf import settings
+from django.test.testcases import TestCase
+from django.utils import timezone
+
+from ..utils.export_data import ReservationUnitExporter
+from .factories import ReservationUnitFactory
+
+
+class ReservationUnitDataExporterTestCase(TestCase):
+    export_dir = Path(settings.BASE_DIR) / "exports" / "reservation_unit_exports"
+    file_name = f"reservation_units__{timezone.now().strftime('%d-%m-%Y')}.csv"
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.reservation_unit = ReservationUnitFactory()
+
+    def tearDown(self) -> None:
+        super().tearDown()
+
+        if self.export_dir.is_dir():
+            shutil.rmtree(self.export_dir)
```

Comment:
Thing to consider; If we would be using file mock here we wouldn't actually need to write anything and cleanup things.